components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -107,23 +139,30 @@ export default class UserLimitAnnouncementBar extends React.PureComponent<Props>
             return null;
         }
 
-        let dismissable = true;
-
-        // If the user limit is less than the current number of users, the banner is not dismissable
-        if (userLimit < analytics!.TOTAL_USERS) {
-            dismissable = false;
-        }
+        const dismissable = this.isDismissable();
 
         return (
             <AnnouncementBar
                 type={dismissable ? AnnouncementBarTypes.ADVISOR : AnnouncementBarTypes.CRITICAL_LIGHT}
                 showCloseButton={dismissable}
                 handleClose={this.handleClose}
-                showModal={() =>
+                showModal={() => {
```

Comment:
Nit: This is kinda too big to be in-line, would prefer it as a external function.

Reply:
Moved this body into proper function